qfieldsync/gui/package_dialog.py

```diff
@@ -145,15 +149,43 @@ def get_export_folder_from_dialog(self):
         # manual
         return self.manualDir.text()
 
+    def get_export_filename_suggestion(self):
+        export_folder = Path(self.get_export_folder_from_dialog())
+        full_project_name_suggestion = export_folder.joinpath(
+            f"{self.project.baseName()}_qfield.qgs"
+        )
+        return str(full_project_name_suggestion)
+
+    def set_export_filename_suggested(self):
+        self.packaged_project_filename.setText(self.get_export_filename_suggestion())
+
     def show_package_page(self):
         self.nextButton.setVisible(False)
         self.button_box.setVisible(True)
         self.stackedWidget.setCurrentWidget(self.packagePage)
 
+    def run_package_project(self):
+        export_packaged_project = Path(self.packaged_project_filename.text())
+
+        if export_packaged_project.exists():
+            reply = QMessageBox.question(
+                self,
+                "File Already Exists",
+                f"'{export_packaged_project.name}' already exists. Do you want to overwrite it?",
```

Comment:
```suggestion
                self.tr("File Already Exists"),
                self.tr('File "{}" already exists. Do you want to overwrite it?').format(export_packaged_project.name),
```
don't forget the translation